package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dellstorage/clarity-react",
-    "version": "1.1.4",
+    "version": "1.1.5",
```

Comment:
we already have 1.1.6 version.  Change this to 1.1.6, so that CI would create 1.1.7

Reply:
done